frontend/package.json

```diff
@@ -152,4 +152,4 @@
     "last 2 versions",
     "not dead"
   ]
-}
+}
```

Comment:
```suggestion
}

```

Reply:
ya I think this and vue changes are remaining, I will quickly commit changes in both the files in all the branches so the PRs will be updated soon